app/helpers.js

```diff
@@ -0,0 +1,13 @@
+const { SORT_ORDER, ASC, DESC } = require('./constants');
+
+exports.paginate = (data, params) => {
+  const { offset, limit } = params;
+  return data.slice(offset, offset + limit);
+};
+
+exports.order = (data, sortKey, sortOrder) => {
+  const sorting = sortOrder === SORT_ORDER ? DESC : ASC;
+  return data.sort((album1, album2) => (album1[sortKey] >= album2[sortKey] ? 1 : sorting));
```

Comment:
May I suggest a little change here? It's a little hard to read this logic. I suggest:

1. Compare values: `x >= y ? 1 : -1`, this will order array ASC
2. If you want to invert the order, you can simply: `(x >= y ? 1 : -1) * -1`. Then it will be DESC

So, you can change your sorting algorithm: 
```javascript
const sorting = (x, y) => (x >= y ? 1 : -1) * constants[sortOrder];
return array.sort((x, y) => sorting(x[field], y[field]))
```